### PR TITLE
docs: Fix some links and descriptions in .cabal file.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/TokTok/hs-msgpack-json/1)

### CODE REVIEW

File: msgpack-json.cabal [hunk 1]
```diff
@@ -14,13 +14,13 @@ build-type:    Simple
 description:
   A Haskell implementation of MessagePack <http://msgpack.org/>
   .
-  This is a fork of msgpack-haskell <https://github.com/msgpack/msgpack-haskell>,
+  This is a fork of msgpack-aeson <https://github.com/msgpack/msgpack-haskell>,
```
Comment (robinlinden) [inline msgpack-json.cabal:17]: Are you sure? :P

edit: ah, I see, msgpack-haskell contains msgpack-aeson as well.

**Review by robinlinden — APPROVED** [review]

File: msgpack-json.cabal (hunk 1 above)
Comment (iphydf) [reply msgpack-json.cabal:17]: Fairly sure? What am I missing?

Comment (iphydf) [reply msgpack-json.cabal:17]: Yep, that repo is a bunch of msgpack packages in one repo. We split it across multiple repos.

Comment (robinlinden) [reply msgpack-json.cabal:17]: My edit was too slow. :(

**Pull request merged**